recipes/moltenvk/all/patches/0002-allow-c++11-std.patch

```diff
@@ -0,0 +1,16 @@
+upstream issue: https://github.com/KhronosGroup/MoltenVK/issues/1474
+
+--- a/MoltenVK/MoltenVK/Commands/MVKCommandBuffer.mm
++++ b/MoltenVK/MoltenVK/Commands/MVKCommandBuffer.mm
+@@ -769,7 +769,10 @@ void MVKCommandEncoder::markTimestamp(MVKTimestampQueryPool* pQueryPool, uint32_
+ 	if (sampPts) {
+ 		for (uint32_t qOfst = 0; qOfst < queryCount; qOfst++) {
+ 			if (mvkIsAnyFlagEnabled(sampPts, MVK_COUNTER_SAMPLING_AT_PIPELINE_STAGE)) {
+-				_timestampStageCounterQueries.push_back({ pQueryPool, query + qOfst });
```

Comment:
Would `_timestampStageCounterQueries.push_back(GPUCounterQuery{ pQueryPool, query + qOfst });` work as well?

Reply:
Didn't try but I don't think it would work. pQueryPool type is a derived class of GPUCounterQuery first member class. It requires at least C++14 to work.